chucks-pick3-web/angular-app/src/app/directives/uppercase-validator.directive.ts

```diff
@@ -0,0 +1,41 @@
+import {NG_VALIDATORS, FormControl, ValidatorFn, Validator} from '@angular/forms';
+import {FormValidationService} from '../services/form-validation.service/form-validation.service';
+import {Directive} from '@angular/core';
+
+// @ts-ignore
+// @ts-ignore
+@Directive({
+  // tslint:disable-next-line:directive-selector
+  selector: '[uppercaseValidator] [ngModel]',
```

Comment:
Let's use camel-case here.  Also rename the file to `upper-case-validator.directive.ts`.

Reply:
Left as camel case and renamed file to upper-case-validator.directive.ts